infra/aws/terraform/registry.k8s.io/accounts.tf

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+resource "aws_iam_user" "registry-k8s-io-access" {
+  name = "${var.prefix}registry-k8s-io-access"
+  path = "/"
+}
+
+resource "aws_iam_access_key" "registry-k8s-io" {
+  user = aws_iam_user.registry-k8s-io-access.name
+}
```

Comment:
How will we protect this access key?
Is Terraform the best way to manage it?

Reply:
These are good questions.

We have one `google_service_account_key` today, FWIW, https://github.com/kubernetes/k8s.io/search?q=google_service_account_key

Reply:
I'm currently unsure how to answer these questions, happy to discuss and figure it out! :smiley: